cookiecutter.json

```diff
@@ -1,8 +1,16 @@
 {
-  "github_org": "sphinx-contrib",
-  "package_name": "The name of the extension package",
-  "project_name": "sphinxcontrib-{{ cookiecutter.package_name}}",
-  "short_description": "A short description of the extension",
-  "author_name": "Your name",
-  "author_email": "Your email"
+  "github_org"           : "sphinx-contrib",
+  "package_name"         : "The name of the extension package",
+  "project_name"         : "sphinxcontrib-{{ cookiecutter.package_name}}",
+  "short_description"    : "A short description of the extension",
+  "author_name"          : "Your name",
+  "author_email"         : "Your email",
+  "version"              : "0.0",
+  "release"              : "0.0.0",
+  "copyright"            : 2019,
+  "documentation_source" : "docs",
+  "documentation_target" : "dist",
+  "index"                : "index",
+  "language"             : "en"
```

Comment:
Not personally a fan of using this style of indentation. It doesn't look like cookiecutter supports YAML, but could we keep this as-is?